scripts/os_configure/install_software_22.bash

```diff
@@ -0,0 +1,251 @@
+
+########################## END BASIC SETUP ##########################
+
+if [[ "$computer_type" != "robot" ]]
+then
+  sudo apt -y install recordmydesktop rdesktop gimp meshlab inkscape pdfposter unrar
+  sudo DEBIAN_FRONTEND=noninteractive apt -y install wireshark
+fi
+
+if [[ $computer_type == "office" ]]
+then
+  # Latex
+  sudo apt -y install kile texlive-full texlive-lang-german kbibtex ktikz
+
+  sudo apt -y install pass
+
+  # Network manager
+  sudo apt -y install network-manager-openvpn network-manager-vpnc network-manager-ssh network-manager-openconnect
+
+  # Cryptography
+  sudo apt -y install kleopatra scdaemon
+
+  # Code optimization and profiling
+  sudo apt -y install valgrind kcachegrind hotspot heaptrack-gui
+  sudo apt -y install linux-tools-generic linux-cloud-tools-generic  # Kernel tools
+
+  sudo apt -y install flac
+
+  sudo add-apt-repository -y ppa:freecad-maintainers/freecad-daily
+  sudo apt update
+  sudo apt -y install freecad-daily spacenavd
+
+  # Nextcloud
+  sudo apt -y install nextcloud-desktop
+
+  # VirtualBox
+  sudo sh -c 'echo "deb [arch=amd64] https://download.virtualbox.org/virtualbox/debian $(lsb_release -sc) contrib" > /etc/apt/sources.list.d/virtualbox.list'
+  wget -q https://www.virtualbox.org/download/oracle_vbox_2016.asc -O- | sudo apt-key add -
+  sudo apt update
+  sudo apt -y install virtualbox-6.1 dkms virtualbox-guest-utils virtualbox-ext-pack
+
+  ## Messgengers
+
+  # Telegram
+  sudo add-apt-repository ppa:atareao/telegram
+  sudo apt update
+  sudo apt -y install -f telegram
+
+  # Hamsket
+  sudo apt -y install libappindicator3-1
+  wget https://github.com/TheGoddessInari/hamsket/releases/download/0.6.2/hamsket_0.6.2_amd64.deb
+  sudo dpkg -i hamsket_0.6.2_amd64.deb
+  rm hamsket_0.6.2_amd64.deb
+
+  # Spotify
+  curl -sS https://download.spotify.com/debian/pubkey_0D811D58.gpg | sudo apt-key add -
+  echo "deb http://repository.spotify.com stable non-free" | sudo tee /etc/apt/sources.list.d/spotify.list
+  sudo apt update
+  sudo apt -y install spotify-client
+
+  # Noson
+  sudo add-apt-repository ppa:jlbarriere68/noson-app
+  sudo apt update
+  sudo apt -y install noson-app
+
+  # KDE-PIM
+  sudo apt -y install kontact korganizer kmail kjots kaddressbook kdepim*
+
+  # Tools
+  sudo apt -y install  krdc
+
+  # Notes-taking
+  sudo add-apt-repository ppa:pbek/qownnotes
+  sudo apt update
+  sudo apt -y install qownnotes
+
+  # Language packs
+  sudo apt -y install language-pack-de language-pack-de-base language-pack-kde-de aspell-de hunspell-de-de hyphen-de wogerman
+  sudo apt -y install language-pack-hr language-pack-hr-base language-pack-kde-hr aspell-hr hunspell-hr hyphen-hr
+
+  ## Development Tools
+
+  # Tracing
+  sudo apt-add-repository ppa:lttng/stable-2.12
+  sudo apt update
+  sudo apt -y install lttng-tools lttng-modules-dkms liblttng-ust-dev
+  sudo apt -y install python3-babeltrace python3-lttng python3-lttngust
+  sudo usermod -aG tracing "$(whoami)"
```

Comment:
useful debugging tool. Should ve moved/sorted in this section.

Yes, can you please sort application according to their use-case?

Reply:
Tried my best.